Description: Display the patch release version 0.18.1 throughout the application
Author: Kurt Kremitzki <kurt@kwk.systems>
Last-Updated: 2019-04-06

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,7 +4,7 @@
 set(PACKAGE_NAME "FreeCAD")
 set(PACKAGE_VERSION_NAME "Vulcan")
 set(PACKAGE_VERSION_MAJOR "0")
-set(PACKAGE_VERSION_MINOR "18")
+set(PACKAGE_VERSION_MINOR "18.1")
 set(PACKAGE_VERSION_PATCH "13500")
 
 set(FREECAD_VERSION "${PACKAGE_VERSION_MAJOR}.${PACKAGE_VERSION_MINOR}")
